codec-http/src/main/java/io/netty/handler/codec/http/ClientCookieEncoder.java

```diff
@@ -15,6 +15,8 @@
  */
 package io.netty.handler.codec.http;
 
+import io.netty.handler.codec.http.cookie.ClientCookieDecoder;
+
```

Comment:
Explicit import protects against javadoc breakage at compile time if a class renamed negligently.
And a simple class name is more comfortable in javadoc..